### PR TITLE
Bl orient741

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient741.xml [hunk 1]
```diff
@@ -53,49 +233,296 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                          </decoNote>
+                        </decoDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1v"></locus>
+                              <desc type="PurchaseNote">A note in Amharic, possibly a purchase note,
+                              of the time of <persName ref="PRS3692egwalas"></persName>.</desc>
+                              <q xml:lang="gez"></q>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient741.xml:364]: am

File: LondonBritishLibrary/orient/BLorient741.xml [hunk 2]
```diff
@@ -53,49 +233,296 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            <locus target="#100rc"></locus>
+                            <desc>Small ornamental band in black.</desc>
+                          </decoNote>
+                        </decoDesc>
                         <additions>
-                            
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1v"></locus>
+                              <desc type="PurchaseNote">A note in Amharic, possibly a purchase note,
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient741.xml:362]: This and a2 are two separate notes?

File: LondonBritishLibrary/orient/BLorient741.xml [hunk 3]
```diff
@@ -53,49 +233,296 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            </item>
+                            <item xml:id="a3">
+                            <locus target="#4r"/>
+                            <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
+                                <placeName ref="INS0101MadhaneAlam"/>.</desc>
+                            <q xml:lang="gez"><sic>አፈወ፡ ርቅ፡ ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔዓም።</placeName></sic>
+                            </q>
+                          </item>
+                            <item xml:id="a4">
+                              <locus target="193r"></locus>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient741.xml:379]: #

File: LondonBritishLibrary/orient/BLorient741.xml [hunk 4]
```diff
@@ -53,49 +233,296 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                            </item>
+                            <item xml:id="e13">
+                              <locus target="#107v"></locus>
+                              <desc><foreign xml:lang="gez"><hi rend="negative">
+                                <sic>ዘወልደ፡ ዮ</sic></hi></foreign> written in negative writing instead of a small
+                              ornamental band at the end of a passage, partially erased. Apparently,it is
+                            the name of the owner, <persName ref="PRS13449WaldaY" role="owner"></persName>.</desc>
+                            </item>
+                            <item xml:id="e14">
+                              <locus target="#1222v"></locus>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient741.xml:494]: 122

File: LondonBritishLibrary/orient/BLorient741.xml [hunk 5]
```diff
@@ -53,49 +233,296 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                              <desc><foreign xml:lang="gez"><hi rend="negative">
+                                ዘወልደ፡ ዮሐንስ</hi></foreign> written in negative writing instead of a small
+                              ornamental band at the end of a passage, partially erased. Apparently,it is
+                            the name of the owner, <persName ref="PRS13449WaldaY" role="owner"></persName>.</desc>
+                            </item>
+                            <item xml:id="e15">
+                              <locus target="#6v #110v #122v #192v"></locus>
+                              <desc>The name of the owner is the name of the owner, <persName ref="PRS13449WaldaY" role="owner"></persName>.</desc>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient741.xml:502]: the name of the owner double

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: LondonBritishLibrary/orient/BLorient741.xml (hunk 2 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient741.xml:362]: Yes, I think they have been written by two different hands.

**Comment by DariaElagina** [comment]
Thank you!

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**